feat: Add qldb.Ledgers resource

roneli opened the pull request:
Closes #626

roneli committed:
feat: Add qldb.Ledgers resoruce

Closes #626

roneli committed:
feat: Add qldb.Ledgers resoruce

Closes #626

roneli committed:
add docs + mock + sql

roneli committed:
add docs + mock + sql

dependabot[bot] committed:
chore(deps): bump github.com/cloudquery/cq-provider-sdk (#670)

Bumps [github.com/cloudquery/cq-provider-sdk](https://github.com/cloudquery/cq-provider-sdk) from 0.8.15 to 0.8.16.
- [Release notes](https://github.com/cloudquery/cq-provider-sdk/releases)
- [Changelog](https://github.com/cloudquery/cq-provider-sdk/blob/main/CHANGELOG.md)
- [Commits](https://github.com/cloudquery/cq-provider-sdk/compare/v0.8.15...v0.8.16)

---
updated-dependencies:
- dependency-name: github.com/cloudquery/cq-provider-sdk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

bbernays committed:
fix: PKs for Virtual MFA Device (#671)

irmatov committed:
feat: Add Backup plans, vaults (#649)

cq-bot committed:
chore: Sync form .github (#673)

amanenk committed:
feat: Ecs tasks (#646)

disq committed:
feat: Drift: More resources (#636)

zagronitay committed:
chore: Added codepipeline::webhook tf (#677)

roneli committed:
add mock tests

+ fix migrations

roneli committed:
add mock tests

+ fix migrations

roneli committed:
chore: Add ledger tf files (#679)

* feat: Add ledger tf files

disq commented on client/client.go:
```diff
@@ -8,6 +8,8 @@ import (
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/qldb"
+
```
rogue newline

disq commented on resources/provider/migrations/postgres/24_v0.10.16.up.sql:
```diff
@@ -87,3 +87,65 @@ ALTER TABLE IF EXISTS aws_mq_broker_users
+     CONSTRAINT aws_qldb_ledger_journal_kinesis_streams_pk PRIMARY KEY(cq_id),
+     UNIQUE(cq_id)
+    );
+CREATE INDEX ON aws_qldb_ledger_journal_kinesis_streams (ledger_cq_id);
```
is this a new thing/case we need to add to the migrations generator?

disq commented on resources/services/qldb/ledgers.go:
```diff
@@ -0,0 +1,345 @@
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "creation_time",
```
not `creation_date_and_time` like the parent resource? inconsistent

roneli replied:
That is how its in the original struct

disq reviewed (approved):
LGTM with nits

zagronitay committed:
fix: Fixed qldb tf backend (#682)

zagronitay committed:
fix: Added tags for codepipeline resources (#684)

zagronitay committed:
feat: Codepipeline webhooks (#678)

* feat: Added Codepipeline

* added migrations

* cr

* docs

* CR

* mods

* CR

* fixed issue

* changed new col name

* feat: Added codepipeline webhooks

* updated aws endpoints

* updated migration files

* fixed integ tests

* fixed migrations

* fixed webhook tags

* lint

roneli committed:
add ignore in tests

roneli committed:
add ignore in tests

disq committed:
feat: Drift: Even more resources (#683)

Co-authored-by: Kemal Hadimli <disq@users.noreply.github.com>

shimonp21 committed:
chore: Update endpoints (#687)

irmatov committed:
chore: Add TF for wafv2 ipsets and regex sets (#676)

irmatov committed:
chore: Add WAF Regional TF resources (#686)

zagronitay committed:
chore: Added ec2 egress-only internet gateway tf (#688)

irmatov committed:
feat: Add WAFv2 IPSets and RegexPatternSets (#675)

Add new ServiceAccountRegionScopeMultiplexer to use scope
to always fetch cloudfront related resources.

disq committed:
fix: Redact "Encoded authorization failure message" (#690)

Co-authored-by: Kemal Hadimli <disq@users.noreply.github.com>

disq committed:
fix: VirtualMFADevices PK (#692)

Co-authored-by: Kemal Hadimli <disq@users.noreply.github.com>

bbernays committed:
feat: Reduce ECS API Calls (#693)

disq committed:
feat: Tag helper (#685)

disq committed:
chore: OldGo compatibility (#699)

Co-authored-by: Kemal Hadimli <disq@users.noreply.github.com>

zagronitay committed:
fix: Updated AWS endpoints (#700)

zagronitay committed:
feat: Added ec2 egress only igw (#689)

spangenberg committed:
feat: Add WebACL Cloudfront fetching (#680)

Fixes #518

roneli committed:
rebase

roneli commented:
Closed for #702
